dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -55,16 +55,16 @@
 public class DFSReadWriteTest extends BaseRemoteTest
 {
     private static final String[] datasets       = { "~benchmark::integer::20kb", "~benchmark::all_types::200kb"};
-    private static final int[]    expectedCounts = { 1250, 5600 };
+    private static final int[]    expectedCounts = { 1250, 1558 };
```

Comment:
was the 5600 record count invalid? @jpmcmu

Reply:
yes, the current file has 1558 recs, I checked on the cluster to confirm.

Reply:
@drealeed the datasets and expectcounts should really be passed in dynamically, but for now our official build environment's file has 5600 recs. 
let's revert, and we'll provide you the ECL we execute to generate the larger dataset